### DISCUSSION TITLE
ressources loaded by html/javascript

### DISCUSSION THREAD

**Discussion by dfroger** [foxyproxy/firefox-extension#150] [Q&A] [answered]
Hi,

I have a `*.example.com` pattern for a proxy.  I visit `https://example.com` and `example.com/index.html` is downloaded via the proxy. `index.html` contains  `<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>`, will `jquery` be downloaded via the proxy?

And what about `GET`, `POST` to `api.foo.com` requests that maybe made by JavaScript from the `index.html`?

Thanks!

**Comment by erosman** [comment, marked as answer]
A webpage may contain many element from different sources and they don't all come under the tab's connection.

Each network connection is separate and will be treated according to the rules.

Therefore, a rule e.g. `*.example.com` will apply to any connection to that domain, `GET`, `POST` etc

`https://code.jquery.com/jquery-3.2.1.min.js` belongs to a different domain and not related to `*.example.com` so unless there is a rule for it, it will not go through the proxy.

**Reply by dfroger** [reply]
Ok, thanks for the answer!

**Reply by adl41** [reply]
Very Nice

**Comment by ericjung** [comment]
@dfroger look at FoxyProxy’s logging tab to see all the URLs loaded by a site. You can then add rules for those if you like
